ert_shared/cli/main.py

```diff
@@ -32,6 +32,17 @@ def run_cli(args):
         if model.hasRunFailed():
             sys.exit(model.getFailMessage())
     else:
+        if args.current_case:
+            ERT.enkf_facade.select_or_create_new_case(args.current_case)
+
+        if (args.mode in ['ensemble_smoother', 'es_mda'] and 
```

Comment:
Can these constant strings be fetched from somewhere (one source of truth). Just in case we rename them in the future, and forget to rename them here.

Reply:
I agree, but I do not have a good suggestions as to from where..

Reply:
to me it would make sense to import them like `from ert.cli import ENSEMBLE_SMOOTHER_COMMAND, ES_MDA_COMMAND`, or something along those lines.